src/consensus/params.h

```diff
@@ -26,13 +26,13 @@ enum DeploymentPos
     // NOTE: Also add new deployments to VersionBitsDeploymentInfo in versionbits.cpp
     DEPLOYMENT_CSV = 0, // bit 0 - deployment of BIP68, BIP112, and BIP113.
     // begin unassigned bits. Rename bits when allocated.
-    DEPLOYMENT_UNASSIGNED_BIT_1,
-    DEPLOYMENT_UNASSIGNED_BIT_2,
-    DEPLOYMENT_UNASSIGNED_BIT_3,
-    DEPLOYMENT_UNASSIGNED_BIT_4,
-    DEPLOYMENT_UNASSIGNED_BIT_5,
-    DEPLOYMENT_UNASSIGNED_BIT_6,
-    DEPLOYMENT_UNASSIGNED_BIT_7,
+    DEPLOYMENT_128MB_BLOCK,
+    DEPLOYMENT_MUL_SHIFT_INVERT,
+    DEPLOYMENT_UNLIMITED_SCRIPT_SIZE,
```

Comment:
Maybe it's just bike shedding but I would call it something `unlimited number of op_codes` or `remove op-codes limit`

Reply:
you are right, the size is still constrained.
DEPLOYMENT_UNLIMITED_SCRIPT_OPCODES,